### DISCUSSION TITLE
Following hashtags

### DISCUSSION THREAD

**Discussion by dogrileycom** [mastodon/mastodon#23045] [General]
When I try and follow a second hashtag it doesn't stay chosen.  

It seems only one hashtag is followable?  I can change my single entry to anything I want but I can't follow 2.

Following #music works, try and add #film it will show selected, doesn't generate an error, but navigate away and back and the selection is unselected.

Has anyone seen this?

Seen in Mastodon iOS client and WebClient

**Comment by FediVideos** [comment]
Are you following this on the website in the single column mode or the multicolumn advanced interface?

The advanced interface uses hashtag pinning which is a bit different from following.
